google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -362,6 +362,18 @@ public LifecycleCondition getCondition() {
       return lifecycleCondition;
     }
 
+    @Override
+    public String toString() {
+      return MoreObjects.toStringHelper(this)
+          .add("action", lifecycleAction.getActionType())
```

Comment:
Could you do a toString() for ActionType? It's missing setStorageClass information in this case.